osu.Game/Screens/Play/Player.cs

```diff
@@ -1043,6 +1044,22 @@ public override bool OnExiting(ScreenExitEvent e)
             return base.OnExiting(e);
         }
 
+        private void saveReplay()
+        {
+            var scoreCopy = Score.DeepClone();
+
+            try
+            {
+                ImportScore(scoreCopy).ConfigureAwait(false);
```

Comment:
Why is this async call fire-and-forgotten? No `await`, no `.ContinueWith()` even?

Reply:
Because I am considering the return type of this method.
I use void for it but code quality said that avoid use void return type for async methods

So it's a bit like my stupid solution.

I will revert it and consider return type